vars/lvUTF.groovy

```diff
@@ -0,0 +1,4 @@
+def call(UTF_path, lvVersion){
+   echo "SRPSM: Running UTF."
+   labviewcli("-OperationName RunUnitTests -ProjectPath \"$WORKSPACE\\$UTF_path\" -JUnitReportPath \"$WORKSPACE\\UTF_path.log\"", lvVersion)
```

Comment:
- [ ] `-JUnitReportPath \"$WORKSPACE\\UTF_path.log\""` logs to a file called `UTF_path.log`. I assume this was meant to be `$UTF_path.log`
- [ ] Add a `-LogFilePath` to somewhere in `$WORKSPACE` like what exists for other lvBuild scripts.

Reply:
Removed UTF